src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1047,7 +1047,7 @@ void GLHEVert::setupTimeVectors()
     while (true) {
         Real64 maxPossibleSimTime = exp(tempLNTTS.back()) * t_s;
         if (maxPossibleSimTime <
-            this->myRespFactors->maxSimYears * numDaysInYear * DataGlobalConstants::HoursInDay * DataGlobalConstants::SecInHour) {
+            this->myRespFactors->maxSimYears * numDaysInYear * Constant::HoursInDay * Constant::SecInHour) {
```

Comment:
I see while (true) above and then numDaysInYear * HoursInDay * SecInHour. Just sayin.

Reply:
Right. For the record, I did this `DataGlobalConstants` to `Constant` change using `grep -l DataGlobalConstants *.cc *.hh | xargs | sed -i s/DataGlobalConstants/Constant/g`. So I have not actually looked at this code other than the deeper changes I made in `OutputReportsTabular` and `EconomicLifeCycleCost`. And I did a lot of those with search-replace also.